app/src/main/res/layout/activity_main.xml

```diff
@@ -35,33 +37,62 @@
         android:id="@+id/server_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_marginTop="400dp"
         android:fontFamily="@font/bitter"
         android:text="@string/behave_as_server"
         app:layout_constraintBottom_toTopOf="@id/client_button"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_chainStyle="packed" />
+        app:layout_constraintTop_toBottomOf="@id/input_username"
+        app:layout_constraintVertical_chainStyle="spread" />
 
     <Button
         android:id="@+id/client_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_margin="20dp"
+        android:layout_margin="10dp"
+        android:layout_marginBottom="0dp"
         android:fontFamily="@font/bitter"
+        android:rotationY="0"
         android:text="@string/behave_as_client"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/server_button" />
+        app:layout_constraintTop_toBottomOf="@+id/server_button" />
 
-    <androidx.fragment.app.FragmentContainerView
+    <androidx.constraintlayout.widget.Guideline
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintGuide_percent="0.7"
+        android:id="@+id/horizontal_guide"
+        android:orientation="horizontal"/>
+
+    <EditText
+        android:id="@+id/input_username"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="-20dp"
+        android:hint="@string/enter_username"
+        android:inputType="text"
+        android:maxLength="5"
+        android:minHeight="48dp"
+        app:layout_constraintBottom_toTopOf="@id/server_button"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/wizards_header"
+        android:autofillHints="@string/autofill_hint_username"
+        />
+
+    <FrameLayout
         android:id="@+id/fragment_container"
+        android:name="at.aau.edu.wizards.ui.lobby.LobbyFragment"
         android:layout_width="0dp"
         android:layout_height="0dp"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.0" />
```

Comment:
we talked about that, please remove and check if the same is applied somewhere else in that file.